src/_data/books.yml

```diff
@@ -1,3 +1,12 @@
+- title: 'Dart: Scalable Application Development'
+  authors:
+    - Davy Mitchell
+    - Sergey Akopkokhyants
+    - Ivo Balbaert
+  publisher: Packt Publishing
+  cover: scalable-app-development.png
+  link: "https://www.packtpub.com/application-development/dart-scalable-application-development"
+  desc: "Master the art of designing for web, client, and server with Google's bold and productive language - Dart."
 - title: Learning Dart [Video]
```

Comment:
The final dash looks funny. Make sure it's an em-dash, and then you're good to go.